packages/desktop-gui/src/project-nav/browsers.jsx

```diff
@@ -17,7 +18,7 @@ export default class Browsers extends Component {
         {this._closeBrowserBtn()}
         <Dropdown
           className='browsers-list'
-          disabled={project.browserState === 'opened' || project.browserState === 'opening'}
+          disabled={project.isBrowserState(Project.BROWSER_OPENING, Project.BROWSER_OPEN, Project.BROWSER_CLOSING)}
```

Comment:
Nice! I like the expressiveness and encapsulation of the conditions over what was there previously.